src/routers/messages.py

```
@@ -25,286 +35,113 @@
+    logger.info(
+        f"Batch of {len(created_messages)} messages created and queued for processing"
+    )
 
-                sentry_sdk.capture_exception(e)
+    return created_messages
 
 
-@router.post("/", response_model=list[schemas.Message])
-async def create_messages_for_session(
+@router.post("/upload", response_model=list[schemas.Message])
+async def create_messages_with_file(
```

Ruff (0.12.2)

Missing return type annotation for public function `create_messages_with_file`

(ANN201)